resources/.eslintrc-standards--annoy-mode.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    "rules": {
+        "no-magic-numbers": ["error", {                          // disallow magic numbers
+            "ignore": [0, 1, -1],
```

Comment:
I was thinking of adding a few more numbers to this list, e.g. 

- 2 for game halves
- 100 for percentage calculations
- 24, 60 and 1000 for time calculations

just because they are so common (in our repos anyway)